翻译项目设置 (#1597)
cocos-creator/creator-docs: en/editor/inspector/index.md

Properties can usually be divided into two main categories, **value types** and **reference types**, depending on where the variables use memory.

### Value type attributes

**Value types** include simple variable types that take up very little memory, such as numbers, strings, booleans, enumerations, etc.

- `Numeric (Number)`: can be entered directly using the keyboard, or the property value can be incremented or decremented by pressing the up and down arrows next to the input box.
- `Vector (Vec2)`: The control of a vector is a combination of two numeric inputs, and the input box is marked with x and y to identify the sub-property name corresponding to each value.
- `String`: Enter a string directly into the text box using the keyboard.
- `Boolean`: Edit in the form of a checkbox, the selected state means the property value is true, the non-selected state means false.
- `Enum`: edit in the form of a drop-down menu, click the enumeration menu, and then select an item from the pop-up menu list to finish modifying the enumeration value.

> - Color: Click on the color property preview box, the **Color Picker** window will pop up, in this window you can use the mouse to directly click on the desired color, or directly enter the specified color in the RGBA color input box below. Clicking anywhere outside the **Color Picker** window will close the window and use the last selected color as the property value. For example, the color picker component.

Reviewer: - `Color`: Click on the color property preview box, the **Color Picker** window will pop up, in this window use the mouse to directly click on the desired color, or directly enter the specified color in the RGBA color input box below. Clicking anywhere outside the **Color Picker** window will close the window and use the last selected color as the property value. For example, the color picker component.

  ! [ui-color](index/ui-color.png)

### Reference type properties

**Reference types** include object objects, such as nodes, components, or assets. They can be selected and assigned by **dragging the node or asset into the properties bar** or by **popping up the asset panel**.

! [assets-panel](index/assets-panel.png)

## Batch operations

> When you need to set **same type** asset properties in batch, you can press Shift in **Asset Explorer** and select multiple assets, **Inspector** will show the number of assets selected and the editable asset properties. Click the **Apply** button at the top right when you are done.

Reviewer: ```suggestion
When needing to set the **same type** asset properties in batch, press Shift in **Asset Explorer** and select multiple assets, **Inspector** will show the number of assets selected and the editable asset properties. Click the **Apply** button at the top right when done.
```

! [multiple-edit](index/multiple-edit1.png)

> Batch modification of node properties is the same. However, if an attribute in the **attribute inspector** displays one of the following states, it means that the attribute has inconsistent attribute values across the multiple assets selected, and you can choose whether to continue to batch modify the attribute as needed.

Reviewer: ```suggestion
Batch modification of node properties is the same. However, if an attribute in the **Attribute Inspector** displays one of the following states, it means that the attribute has inconsistent attribute values across the multiple assets selected, and choose whether to continue to batch modify the attribute as needed.
```

> - The check box displays **Grey**

Reviewer: ```suggestion
- The check box displays **Grey**.
```

> - The input box displays **-**

Reviewer: ```suggestion
- The input box displays **-**.
```

> - The selection box displays **blank**

Reviewer: ```suggestion
- The selection box displays **blank**.
```

> **Note**.
> 1. Bulk setting operations are not currently supported for Material assets.
> 2. **Assets of different types** can be selected at the same time, but they do not support setting properties in bulk.

## Edit Prefab node properties

> The Prefab node functions in the **Inspector** top toolbar include: unassociate, locate asset, restore from asset, and update to asset. For details, see [Prefab Assets (Prefab)](... /... /asset/prefab.md).

Reviewer: ```suggestion
The Prefab node functions in the **Inspector** top toolbar include: disassociate, locate asset, restore from asset, and update to asset. For details, please refer to the [Prefab Assets (Prefab)](... /... /asset/prefab.md) documentation.
